spec/models/tslr_claim_spec.rb

```diff
@@ -291,4 +298,46 @@
       end
     end
   end
+
+  describe "#complete!" do
+    context "when claim is valid" do
+      let(:tslr_claim) { create(:tslr_claim, :valid_and_eligible) }
+
+      it "sets completed_at to now" do
+        freeze_time do
+          tslr_claim.complete!
+
+          expect(tslr_claim.reload.completed_at).to eq Time.zone.now
+        end
+      end
+
+      context "and the claim has already been completed" do
```

Comment:
I'm not sure that this is something we want to set in stone as required behaviour by capturing it as a spec. Are there circumstances where we would want the user to be able to re-"complete" their claim and change the timestamp for when it was completed? I can think of a reason why we would probably want to prevent this, i.e. it could potentially mess around with the batching process if the timestamp for a claim changed after it was "completed".

Reply:
I took this spec as more informative of the actual behaviour than as a requirement. We decided not to worry about preventing resubmission at this point, but do you think it needs addressing now?

Reply:
Yeah, the problem with a spec is that it signals to someone reading it that the behaviour is important enough to fail the build if the behaviour is removed. It makes it harder to refactor the underlying code without knowing which specs are hard routes and which ones are simply informative.

Reply:
Ok, I will remove this spec.